docs/marketplace/add_to_marketplace/add_item_by_type/add_sidecar.md

```diff
@@ -0,0 +1,319 @@
+---
+id: add_sidecar
+title: Create a sidecar
+sidebar_label: Create a sidecar
+---
+
+[sidecar containers](/development_suite/api-console/api-design/microservice-containers.md#sidecar-containers) are secondary utility containers running side by side with the [main container](/development_suite/api-console/api-design/microservice-containers.md#main-container) in the same host.
+
+sidecars can be added to your Project from the Marketplace or directly specifying a docker image.
+
+Sometimes you may want to add your sidecar to multiple Services. Adding the sidecar from docker image and setting up the correct values for the sidecar configuration for each Service might be a little tricky and error prone.
+
+To ease this process, Mia-Platform lets you upload your sidecar resource definitions to the Marketplace. This way you can define once the correct sidecar configuration and then replicate the same configuration each time you add the sidecar on your Projects.
+
+### Define the sidecar resource
+
+Create a `my-sidecar.json` file and fill it with the sidecar resource definition.
+
+```json
+{
+  "name": "My sidecar", 
+  "itemId": "my-sidecar",
+  "description": "A very simple sidecar",
+  "type": "sidecar",
+  "tenantId": "my-tenant-id",
+  "documentation": {
+    "type": "externalLink",
+    "url": "https://github.com/my-sidecar/docs"
+  },
+  "resources": {
+    "name": "my-sidecar",
+    "dockerImage": "ghcr.io/my-sidecar",
+    "containerPorts": [
+      {
+        "from": 80,
+        "name": "http",
+        "protocol": "TCP",
+        "to": 5000
+      }
+    ],
+  },
+}
+```
+
+In this example, the configuration defines a very simple sidecar resource by specifying only the docker image of the sidecar application and exposing it to the `5000` port of the container. To define a more specific configuration with `ConfigMaps`, `Probes` or `EnvironmentVariables` too just follow the `json-schema` below.
+
+<details>
+<summary>Click to view the full JSON Schema for a sidecar</summary>
+<p>
+
+```json
+{
+  "type": "object",
```

Comment:
The schema of marketplace metadata maybe could be omitted

Reply:
or at least it should be specify that the type should be equal to `sidecar`

Reply:
we cannot just omit it, otherwise one does not know how to configure it.
i see that other "add X to marketplace" pages specify in detail parameters with schemas or more detailed examples too.

to me it seems more valuable to provide somewhere (maybe not here but where else?) the full schema other than an example with everything configured. I find it a bit too complex to understand if i need to customize it with my config

anyway i made more clear the `type: "sidecar"` :+1: